android/expoview/build.gradle

```diff
@@ -96,13 +100,16 @@ task generateDynamicMacros(type: Exec) {
   }
 }
 preBuild.dependsOn generateDynamicMacros
+// REMOVE_WHEN_VERSIONING_TO_HERE
 
 dependencies {
   api fileTree(dir: 'libs', include: ['*.jar'])
-  api 'com.android.support:multidex:1.0.1'
+  api 'com.android.support:multidex:1.0.3'
 
+  // WHEN_VERSIONING_REPLACE_WITH_DEPENDENCIES
+
+  // REMOVE_WHEN_VERSIONING_FROM_HERE
```

Comment:
very tiny nit: can we change this so it matches the syntax of the other comments (`WHEN_VERSIONING_REMOVE_FROM_HERE`)? 😅